data/mods/Magiclysm/worldgen/mine_balrog.json

```diff
@@ -120,7 +120,7 @@
               { "item": "armguard_lightplate", "prob": 60, "damage": 4 },
               { "item": "gloves_plate", "prob": 40, "damage": 4 },
               { "item": "legguard_lightplate", "prob": 60, "damage": 4 },
-              { "item": "armor_lightplate", "prob": 70, "damage": 4 },
+              { "item": "armor_junk_lightplate", "prob": 70, "damage": 4 },
```

Comment:
this might deserve some real armor instead of junkplate.

I imagine in the magic setting you could even make it one of your new suits

Reply:
@KorGgenT Any thoughts on this? I'm removing all the old plate armor and replacing it with junk plate (budget steel versions of existing plate) but it would make sense to have a high value reward here. Would it be appropriate to have a suit of the new plate spawn on this list? If so, should it be highest quality?